cetmix_tower_server/security/cx_tower_variable_value_security.xml

```diff
@@ -1,24 +1,45 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
 
-    <record id="cx_tower_variable_value_rule_group_manager_access" model="ir.rule">
-        <field name="name">Tower variable value: manager access rule</field>
-        <field name="model_id" ref="model_cx_tower_variable_value" />
-        <field name="domain_force">['|', ('is_global', '=', True),
-            ('server_id.message_partner_ids', 'in', [user.partner_id.id])]</field>
-        <field
+        <!-- user -->
+        <record id="cx_tower_variable_value_rule_group_user_access" model="ir.rule">
+            <field name="name">Tower Variable Value: User Access Rule</field>
+            <field name="model_id" ref="model_cx_tower_variable_value" />
+            <field name="domain_force">
+                ['|',
+                 ('is_global', '=', True),
+                 '&amp;',
+                 ('variable_id.access_level', '=', '1'),
+                 ('server_id.message_partner_ids', 'in', [user.partner_id.id])
+                ]
+            </field>
+            <field name="groups" eval="[(4, ref('cetmix_tower_server.group_user'))]" />
+        </record>
+
+        <!-- manager -->
+        <record id="cx_tower_variable_value_rule_group_manager_access" model="ir.rule">
+            <field name="name">Tower Variable Value: Manager Access Rule</field>
+            <field name="model_id" ref="model_cx_tower_variable_value" />
+            <field name="domain_force">
+                ['|',
+                 ('is_global', '=', True),
+                 '&amp;',
+                 ('variable_id.access_level', 'in', ['2']),
```

Comment:
Manager also has access to records with access level less than 2. Check other models rules for example.